qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -433,11 +433,12 @@ <h4>
       {% end %}
 
       <span id="prep-name-span">{{name}}</span> - ID {{prep_id}} ({{data_type}})
-      {% if user_level in ('admin', 'wet-lab admin') and creation_job is not None %}
-          <a class="btn btn-default" download="{{creation_job_filename}}" href="data:text/plain;charset=utf-8,{{creation_job_filename_body}}"><span class="glyphicon glyphicon-download-alt"></span> SampleSheet</a>
-          {% if creation_job_artifact_summary is not None %}
-            <a class="btn btn-default" target="_blank" href="{% raw qiita_config.portal_dir %}/artifact/html_summary/{{creation_job_artifact_summary}}"><span class="glyphicon glyphicon-download-alt"></span> Creation Job Output</a>
+
+      {% if editable and creation_job is not None and creation_job_artifact_summary is not None %}
```

Comment:
This change to the html template is to support the results of the SPP new results `Human Filter & QC existing Prep` and partially address: https://github.com/qiita-spots/qiita/issues/3477.